src/app.module.css

```diff
@@ -10,7 +10,7 @@
   display: grid;
   grid-template-columns: repeat(auto-fit, 14rem);
   grid-gap: 2rem;
-  place-content: center;
+  place-content: flex-start;
```

Comment:
@bsonntag WDYT about this? This put the ui uglier but it is more user friendly.

Reply:
I dunno... I'm planning on adding a "settings" feature, to allow the user to change things like the background color. This could be one such setting... I'll keep it centered for now.